0.4/examples/invalid/mismatch_axes_units.json

```diff
@@ -0,0 +1,38 @@
+{
+    "multiscales": [
+        {
+            "axes": [
+                {
+                    "name": "t",
+                    "type": "time",
+                    "units": "micrometer"
```

Comment:
Similar to the comment before this is technically valid if we have a SHOULD in the units.

Reply:
Agreed, although as discussed yesterday, this is typically a real-world example where I find the consequences of the `SHOULD` requirement nonsensical. I'll open a corresponding issue

Reply:
> Agreed, although as discussed yesterday, this is typically a real-world example where I find the consequences of the SHOULD requirement nonsensical.

Yeah, I agree that the consequences here are weird, but probably better to open an issue and not block the current release.

Reply:
(and at least this would fail if we had a strict schema)